client/src/components/Chat.jsx

```diff
@@ -40,6 +40,7 @@ const Chat = () => {
         messageID: null,
     });
     const [isQuoteReply, setIsQuoteReply] = useState(false)
+    const [message, setMessage] = useState('');
```

Comment:
this doesn't concern your issue either, unless you are using it to calculate the height, which i don't see

Reply:
Yes it was used to calculate the text input  height through the message to setMessage(e.target.value)